### ISSUE TITLE
Text that is both hyperlinked and formatted as inline code does not pass accessibility checks

### ISSUE THREAD

**Issue by phrdang** [berkeley-cdss/berkeley-class-site#63]
Having text like this does not have sufficient color contrast:

```md
[this is some `text`](https://example.com)
```

Here is an example failed CI run: https://github.com/berkeley-cdss/docs/actions/runs/15382832303/job/43276249772?pr=22

**Comment by pancakereport** [comment]
I just checked and your example is not bringing up any a11y violations from what I see. Probably fixed by Brandon in #70 please reopen or file another issue if you find that this is still a problem.
